policy/modules/system/systemd.te

```diff
@@ -1294,6 +1301,7 @@ systemd_read_efivarfs(systemd_userdbd_t)
 
 allow systemd_sleep_t self:capability sys_resource;
 dontaudit systemd_sleep_t self:capability sys_ptrace;
+allow systemd_sleep_t self:lockdown integrity;
```

Comment:
Please add a short comment above this one as well for future reference.

Reply:
A oneliner added.